test/test.config.js

```diff
@@ -7,6 +7,6 @@ const path = require('path');
 config.mocha.tests.push(path.join(__dirname, 'mocha'));
 
 const cfg = config['vc-verifier'];
-cfg.ledgerHostname = 'genesis.veres.one.localhost:42443';
+cfg.mode = 'test';
```

Comment:
I thought we succeeded in eliminating this all together?  This would be configured via bedrock-did-io and the default is already `test`.

https://github.com/digitalbazaar/bedrock-did-io/blob/main/lib/config.js#L14

Reply:
Removed here 0dd9b44